msft_Azure_IoT/msft_Azure_IoT.c

```diff
@@ -1249,14 +1404,51 @@ void prvmcsft_Azure_TwinTask( void * pvParameters )
 
 				break;
 
+#ifdef THINGSPACE_LOCATION_ENABLE
+			case AZURE_SM_LOCATION_UPDATE:
+				vTaskDelay(pdMS_TO_TICKS(1000));
+
+				if (bIsStartUpPhase) {
+					eAzure_SM_Task = AZURE_SM_CONNECT_TO_DPS;
+
+					// Configure HTTPS
+					CellIoT_lib_setTLSSecurityProfileCfg(3, 2, "", 0, 5, -1, -1, NULL, NULL, NULL, 3000);
```

Comment:
Are there enumerations for each of these numbers?  It would make the code more readable there were descriptive constants instead of magic numbers.

Reply:
Good catch. I'll add in some #defs for those magic numbers.